src/data/requestFiltering.js

```diff
@@ -25,10 +44,10 @@ export function populateRequestFilterFrom(queryStringData, me, events) {
       value: queryStringData.creator,
     },
     destination: {
-      active: !! queryStringData.destination,
-      label: 'To: ' + queryStringData.destination,
-      isMatch: request => stringIsIn(queryStringData.destination, request.destination.description),
-      value: queryStringData.destination,
+      active: !! queryStringData.toDescription,
+      label: 'To: ' + queryStringData.toDescription,
```

Comment:
@Schparky I'll let you pick which way to go here ;-) 
```suggestion
      label: `To: ${queryStringData.toDescription}`,
```